packages/fhir-sanitize-param/index.js

```diff
@@ -72,142 +72,212 @@ function findMatchForConfig(name, params) {
 }
 
 /**
- * @function parseAndSanitizeToken
- * @description Given a token, parse it into a code and system
- * @param {String} token - Token to be parsed
- * @return {Token}
+ * @function parseArguments
+ * @description Parse only arguments needed for this type of request
+ * @param {Express.req} req - Request from an express server
+ * @return {Object} - Arguments object
  */
-function parseAndSanitizeToken(token) {
-	let chunks = token.split('|');
-	let system = chunks.length === 1 ? '' : chunks[0];
-	let code = chunks.length === 1 ? token : chunks[1];
+function parseArguments(req) {
+	let args = {};
+	// For GET requests, merge request query
+	if (req.method === 'GET') {
+		args = Object.assign(args, req.query);
+	}
+	// For PUT and POST requests, merge request body
+	if (/PUT|POST/.test(req.method)) {
```

Comment:
Could we make this an else if? I think the issue we were seeing with people misusing Postman could be solved by not mixing GET and POST/PUT params